.github/workflows/markdownlint-problem-matcher.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Can you point me to some doc for this json configuration?

Reply:
@Evangelink For the file you're currently commenting on, I don't know unfortunately. The information I have on it that it makes GitHub prints out the failures in the files changed tab. which is easier than checking the logs.

Reply:
@Evangelink  Found some doc:

https://github.com/actions/toolkit/blob/7ae226462403d6bceb659e29211bc8e812843503/docs/problem-matchers.md

Reply:
Thanks!